milestones/3_human_brain_mapping_journal_submission/step_1_generate_features/ccSBATCH.m

```diff
@@ -11,11 +11,11 @@
         % Nodes, cpus per task, GPUs per node
         nodes = 1;
         ntasksPerNode = 1;  % Must stay at 1
-        cpusPerTask = 40;   % To allow speedup on entire nodes
+        cpusPerTask = 15;   % To allow speedup on entire nodes
```

Comment:
Why are you using 40 CPU for the generate_aec_graphs , but only 15 here?

Reply:
Good questions, I was trying to get priority on the cluster so I reduced the amount of cpu needed so that the slurm scheduler will prioritze me. The generate features isn't taking too long to run anyway so there was a trade of to be made here. This is flexible.